File: solutions/go/two-fer/1/two_fer.go
package twofer

import "fmt"

// ShareWith function print composible string
func ShareWith(name string) string {
	if len(name) > 0 {
        return fmt.Sprintf("One for %s, one for me.", name)
    }

    return fmt.Sprintf("One for you, one for me.")
}

File: solutions/c/two-fer/1/two_fer.c
#include "two_fer.h"


void two_fer(char *buffer, const char *name) {
    if (name == NULL)
    {
        snprintf(buffer, BUFFER_SIZE, "One for you, one for me.");
    } else {
        snprintf(buffer, BUFFER_SIZE, "One for %s, one for me.", name);
    } 
}
